recipes/mss/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = "1.2.0" %}
+
+package:
+  name: mss
+  version:  {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    entry_points:
+      - mss = mslib.msui.mss_pyui:main
+      - mswms = mslib.mswms.mswms:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl >=7.45.0
+    - cycler >=0.10.0
+    - execnet >=1.3.0
+    - freetype >=2.5.5
+    - functools32 >=3.2.3.2
+    - hdf5 >=1.8.15.1
+    - libpng >=1.6.22
```

Comment:
Having `>=` is better than `==` b/c it is less strict, but I believe not all of those are true. Also, `libpng`, `hdf5 (and a few others) needs special pinnings too ensure consistent in our stack (meaning that a newer version with a different `ABI` won't break the package). Here `libpng` should be:

`libpng >=1.6.21,<1.7`